scene/resources/texture.cpp

```diff
@@ -438,12 +449,17 @@ void PortableCompressedTexture2D::create_from_image(const Ref<Image> &p_image, C
 	switch (p_compression_mode) {
 		case COMPRESSION_MODE_LOSSLESS:
 		case COMPRESSION_MODE_LOSSY: {
+			bool lossless_force_png = GLOBAL_GET("rendering/textures/lossless_compression/force_png") ||
```

Comment:
Is this necessary to fix anything? I wasn't involved in the decision to switch to WebP, but I know that the contributors who did agonized over the decision and concluded that it was better to go all in on WebP. Accordingly, I would stick to WebP if at all possible.

Reply:
This fragment of code is copied as is from [resource_importer_texture.cpp](https://github.com/godotengine/godot/blob/95c8216923fc9353ac50f469836ae149cb196e83/editor/import/resource_importer_texture.cpp#L247).

Is it worth moving this common fragment into a separate function so that both of these classes can use it?

Reply:
Ah, in that case, it is probably fine as is.